cypress/e2e/login.cy.js

```diff
@@ -1,11 +1,80 @@
 /// <reference types="cypress" />
 
+const baseURL = 'https://the-internet.herokuapp.com/login';
+const username = 'tomsmith';
+const password = 'SuperSecretPassword!';
+const invalidUsername = 'trustMe';
+const invalidPassword = '30914';
+
 describe('Login page', () => {
   beforeEach(() => {
-    
+    cy.visit(baseURL)
+  });
+
+  it('Should log in with valid credentials', () => {
+    cy.get('#username')
+    .clear()
+    .type(username);
```

Comment:
```suggestion
  it('Should log in with valid credentials', () => {
    cy.get('#username')
      .clear()
      .type(username);
```
check the code style